G/GoogleCodeSearch/Compat.toml

```diff
@@ -1,5 +1,9 @@
-[0]
+["0-0.1.0"]
 BinaryProvider = "0"
 HTTP = "0"
 JSON = "0"
 julia = "1"
+
+["0.1.1-0"]
+HTTP = "0.8-0"
+julia = "1.0"
```

Comment:
Why isnt this just `julia = "1"` together with the `julia` from L5 @StefanKarpinski ?

Reply:
Nvm, guess because of https://github.com/tanmaykm/GoogleCodeSearch.jl/blob/45f0cf24391686bd8fee596fafcca123dc123adb/Project.toml#L20
Does this package not work on Julia 1.1?

Reply:
ah! my bad. will update